Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Process Create Event ASIM parser for Sysmon
-  Version: '0.3.2'
-  LastUpdated: Feb 23, 2022
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
done